triggermail_templates.py

```diff
@@ -55,6 +55,7 @@ def get_extra_params(self):
 
     def run(self, edit):
         self.url = get_url(self.settings) + self.COMMAND_URL
+        use_pertinent_load = self.settings.get('pertinent_load', False)
```

Comment:
I think this variable only gets used once; any reason to not just inline it in the if statement below? Or define it immediately before it gets used? It will be easier to understand what controls this if statement when reading it, if the two parts are close together.